src/Ads/AccountService.php

```diff
@@ -221,6 +227,52 @@ public function get_billing_status(): array {
 		];
 	}
 
+	/**
+	 * Gets the ads account access status.
+	 *
+	 * @return array
+	 * @throws Exception When email is not present or has no account access.
+	 */
+	public function get_ads_account_has_access() {
+		// Check if ads id is present.
+		$ads_id = $this->options->get_ads_id();
+
+		if ( empty( $ads_id ) ) {
```

Comment:
`get_ads_id` always returns an ID so no need to do a `empty` check. `$ads_id` is also not used anywhere else in this function so we can save ourselves a line of code.

```suggestion
		// Check if ads id is present.
		if ( ! $this->options->get_ads_id() ) {
```